README.md

```diff
@@ -1,133 +1,57 @@
 
-+ [cpplint](https://github.com/cpplint/cpplint) for linting code
-+ [FLAMEGPU2-visualiser](https://github.com/FLAMEGPU/FLAMEGPU2-visualiser) dependencies
-  + [SDL](https://www.libsdl.org/)
-  + [GLM](http://glm.g-truc.net/) *(consistent C++/GLSL vector maths functionality)*
-  + [GLEW](http://glew.sourceforge.net/) *(GL extension loader)*
-  + [FreeType](http://www.freetype.org/)  *(font loading)*
-  + [DevIL](http://openil.sourceforge.net/)  *(image loading)*
-  + [Fontconfig](https://www.fontconfig.org/)  *(Linux only, font detection)*
+![sample/figures/visualisation/screenshot.png](sample/figures/visualisation/screenshot.png)
 
-For the plotting script, Python >= 3.6 is required, with python module dependencies listed in `requirements.txt`.
+## Building and Running the Benchmark
 
-## Building with CMake
-
-Building via CMake is a three step process, with slight differences depending on your platform.
-
-1. Create a build directory for an out-of tree build
-2. Configure CMake into the build directory
-    + Using the CMake GUI or CLI tools
-    + Specifying build options such as the CUDA Compute Capabilities to target, the inclusion of Visualisation or Python components, or performance impacting features such as `SEATBELTS`.
-3. Build compilation targets using the configured build system
-
-### Linux
-
-To configure and build the `submodel_benchmark` binary on linux for benchmarking using Volta generation (SM 70) GPU(s), configure as a Release build with SEATBELTS disabled via :
-
-```bash
-# Create the build directory and change into it
-mkdir -p build && cd build
-
-# Configure CMake from the command line passing configure-time options. 
-cmake .. -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DCUDA_ARCH=70 
-
-# Build the target(s)
-cmake --build . --target submodel_benchmark -j 8
-```
-
-Alternatively to configure for visualisation purposes, enable the `VISUALISATION` CMake option:
+Detail of dependencies and the `cmake` build process are described in full in the [FLAMEGPU2-example-template Repo](https://github.com/FLAMEGPU/FLAMEGPU2-example-template) and are not repeated here. The benchmark should be built with seatbelts off (e.g. `-DSEATBELTS=OFF` passed to the `cmake` configuration step) to disable additional run-time checks and optionally disabling Python Swig support which is not needed for this model(-DBUILD_SWIG_PYTHON=OFF ). E.g. for Volta (`SM_70`) GPUs under Linux.
 
 ```bash
-# Create the build directory and change into it
-mkdir -p build && cd build
-
-# Configure CMake from the command line passing configure-time options. 
-cmake .. -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DCUDA_ARCH=70 -DVISUALISATION=ON
-
-# Build the target(s)
-cmake --build . --target submodel_benchmark -j 8
+# Configure 
+cmake . -B build -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DBUILD_SWIG_PYTHON=OFF -DCUDA_ARCH=70
+cmake --build build -j`nproc` 
+make
```

Comment:
`make` is not required and incorrect in this context, `cmake --build` does this

Reply:
done